docs/sources/tempo/metrics-generator/_index.md

```diff
@@ -65,3 +65,11 @@ To learn more about the endpoint configuration, refer to the [Metrics-generator]
 Writing interval can be controlled via `metrics_generator.registry.collection_interval`.
 
 When multi-tenancy is enabled, the metrics-generator forwards the `X-Scope-OrgID` header of the original request to the `remote_write` endpoint. This feature can be disabled by setting `remote_write_add_org_id_header` to false.
+
+## Native histograms
+
+The metrics-generator supports the ability to produce native histograms, for
+high-resolution data. Users must [update the receiving endpoint](https://grafana.com/docs/mimir/latest/configure/configure-native-histograms-ingestion/) to ingest native
```

Comment:
@zalegrala Clarifying question here. Does metrics-generator require a Prometheus or MImir endpoint? Asking because I wasn't sure if Tempo was importing this or not.

Reply:
Correct, Tempo metrics do require Mimir or Prometheus.

Reply:
I've added a link to where we talk about native histograms.

Reply:
Thank you